webapp/src/components/modals/create_update_issue/create_update_issue.jsx

```diff
@@ -47,11 +55,22 @@ export default class CreateIssueModal extends PureComponent {
     }
 
     componentDidUpdate(prevProps) {
-        if (this.props.post && !prevProps.post) {
+        if (this.props.post && !prevProps.post && !this.props.title) {
             this.setState({issueDescription: this.props.post.message}); //eslint-disable-line react/no-did-update-set-state
-        } else if (this.props.channelId && (this.props.channelId !== prevProps.channelId || this.props.title !== prevProps.title)) {
-            const title = this.props.title.substring(0, MAX_TITLE_LENGTH);
-            this.setState({issueTitle: title}); // eslint-disable-line react/no-did-update-set-state
+        } else if (this.props.channelId && (this.props.channelId !== prevProps.channelId || this.props.title !== prevProps.title || this.props.description !== prevProps.description || this.props.assignees !== prevProps.assignees || this.props.labels !== prevProps.labels || this.props.milestoneTitle !== prevProps.milestoneTitle || this.props.milestoneNumber !== prevProps.milestoneNumber)) {
+            if (this.props.assignees) {
+                this.setState({assignees: this.props.assignees}); // eslint-disable-line react/no-did-update-set-state
+            }
+            if (this.props.labels) {
+                this.setState({labels: this.props.labels}); // eslint-disable-line react/no-did-update-set-state
+            }
+            this.setState({milestone: { // eslint-disable-line react/no-did-update-set-state
+                value: this.props.milestoneNumber,
+                label: this.props.milestoneTitle,
+            }});
+            this.setState({issueDescription: this.props.description}); // eslint-disable-line react/no-did-update-set-state
```

Comment:
we dont use setState in componentDidUpdate lifecycle method. Please explore the reason if you don't know and try to find a other method to do this

Reply:
I think we can do that but with a condition otherwise there will be rendering in infinite loop.

Reply:
rather putting on every makes a block for eslint-disable-line